airflow/providers/amazon/aws/utils/eks_get_token.py

```diff
@@ -58,13 +57,7 @@ def main():
     eks_hook = EksHook(aws_conn_id=args.aws_conn_id, region_name=args.region_name)
     access_token = eks_hook.fetch_access_token_for_cluster(args.cluster_name)
     access_token_expiration = get_expiration_time()
-    exec_credential_object = {
-        "kind": "ExecCredential",
-        "apiVersion": "client.authentication.k8s.io/v1alpha1",
-        "spec": {},
-        "status": {"expirationTimestamp": access_token_expiration, "token": access_token},
-    }
-    print(json.dumps(exec_credential_object))
+    print(f"expirationTimestamp: {access_token_expiration}, token: {access_token}")
```

Comment:
Printing the timestamp and token on separate lines would make your grepping/regex a little easier in the wrapper script. But just a suggestion, no need to change if you're happy with how it's implemented now.